src/AuthenticationPortal.Web/Middlewares/ExceptionHandlerMiddleware .cs

```diff
@@ -0,0 +1,56 @@
+﻿using AuthenticationPortal.Contracts;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace AuthenticationPortal.Web
+{
+    public class CustomExceptionMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public CustomExceptionMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            try
+            {
+                await _next.Invoke(context);
+            }
+            catch (Exception ex)
+            {
+                await HandleExceptionAsync(context, ex);
+            }
+        }
+
+        private static Task HandleExceptionAsync(HttpContext context, Exception exception)
+        {
+            var response = context.Response;
+            var customException = exception as BaseException;
+            int StatusCode = (int)HttpStatusCode.InternalServerError;
+            CustomErrorResponse customErrorResponse = new CustomErrorResponse()
+            {
```

Comment:
all custom error messages should be driven from resx file impl

Reply:
Done